demo/go.mod

```diff
@@ -3,12 +3,12 @@ module demo
 go 1.19
 
 require (
-	github.com/google/uuid v1.3.1
-	github.com/open-feature/go-sdk v1.7.0
-	github.com/spotify/confidence-openfeature-provider-go v0.1.3
+	github.com/google/uuid v1.6.0
+	github.com/open-feature/go-sdk v1.10.0
+	github.com/spotify/confidence-openfeature-provider-go v0.1.7
```

Comment:
We should figure out a way for this demo app to depend directly on the source instead of a published version in order to safely be able to test the provider.

Reply:
To my understanding, the `replace` line below should work for that

Reply:
ah, nice! 👍